docs(readme): rewrite intro and switch metalbear.co links to metalbear.com (#4272)
metalbear-co/mirrord: README.md

</div>

[![Community Slack](https://img.shields.io/badge/Join-e5f7f7?logo=slack&label=Community%20Slack)](https://metalbear.com/slack)
[![Github CI](https://github.com/metalbear-co/mirrord/actions/workflows/ci.yaml/badge.svg)](https://github.com/metalbear-co/mirrord/actions/workflows/ci.yaml)
[![License](https://img.shields.io/badge/license-MIT-green)](https://github.com/metalbear-co/mirrord/blob/main/LICENSE)
[![GitHub release (latest SemVer)](https://img.shields.io/github/v/release/metalbear-co/mirrord)](https://github.com/metalbear-co/mirrord/releases)

> [![Twitter Follow](https://img.shields.io/twitter/follow/metalbearco?style=social)](https://twitter.com/metalbearco)

Reviewer: Related, I noticed yesterday that this link is wrong :)

> [![Twitter Follow](https://img.shields.io/twitter/follow/metalbear?style=social)](https://twitter.com/metalbear)

Reviewer 2: Do we want to change this to x?

```suggestion
[![X Follow](https://img.shields.io/twitter/follow/metalbear?style=social)](https://x.com/metalbear)
```

[mirrord runs your local process inside a live Kubernetes cluster](https://metalbear.com/mirrord/).

> It works the same way for a developer in an IDE and for an AI coding agent (Claude Code, Cursor, Codex, Copilot, Windsurf):

Reviewer 2: nit:
```suggestion
It works the same way for developers and for AI coding agents (Claude Code, Cursor, Codex, Copilot, Windsurf):
```

> your code stays on your machine, but mirrord routes its traffic, files, and environment through a target pod in the cluster.

Reviewer 2: would "runs" be more informative than "stays" here? also "routes its [...] environment" feels unclear, we mean to talk about environment variables here right? 

```suggestion
your code runs on your machine, but mirrord routes its traffic, files, and environment through a target pod in the cluster.
```

> That covers both halves of the loop. Read live cluster context while writing the code (real env vars, real service responses, real queue contents)

Reviewer 2: ```suggestion
That covers both halves of the software development loop. Read live cluster context while writing the code (real env vars, real service responses, real queue contents)
```
